data/OperatingSystemApp/controlpanel.ui

```diff
@@ -136,7 +136,7 @@
       </packing>
     </child>
     <child>
-      <object class="Lockscreen" id="wobblyLock">
+      <object class="WobblyLockscreen" id="wobblyLock">
```

Comment:
This will make this file impossible to load in Glade unless you add it to the Glade catalog too.